includes/class-wc-gateway-ppec-client.php

```diff
@@ -698,6 +699,59 @@ protected function _get_details_from_order( $order_id ) {
 		return $details;
 	}
 
+	/**
+	 * Get PayPal shipping address from customer.
+	 *
+	 * @return array Address
+	 */
+	protected function _get_address_from_customer() {
+		$customer = WC()->customer;
+
+		$shipping_address = new PayPal_Address;
+
+		$old_wc = version_compare( WC_VERSION, '3.0', '<' );
+
+		if ( $customer->get_shipping_address() || $customer->get_shipping_address_2() ) {
```

Comment:
Do we need to check the 2nd line in the shipping address here? Is there a scenario where a valid address only has something on the 2nd line?

Reply:
👍 Fair enough